tutorial/README.md

```diff
@@ -20,10 +20,14 @@ After completing the getting started section you should be connected to an AKS c
 Clone this repository locally using whatever technique you prefer. Example:
 
 ```sh
-git clone git@github.com/azure/azure-oam-solution.git
+git clone git@github.com:Azure/azure-oam-solution.git
```

Comment:
This literally has no effect right? 

I'm surprised that Azure hasn't renamed their org to lower case.... The rest of `microsoft` and `dotnet` did :)

Reply:
Casing has no effect, the `:` does make a difference.